libs/common/src/types/content-announcement/types.gen.ts

```diff
@@ -0,0 +1,66 @@
+// This file is auto-generated by @hey-api/openapi-ts
+
+export enum AnnouncementType {
+    Tombstone = 0,
+    Broadcast = 2,
+    Reply = 3,
+    Reaction = 4,
+    Profile = 5,
+    Update = 6,
+    PublicFollows = 113
+}
+
+export type AnnouncementResponse = {
+    /**
+     * An optional identifier for the request, may be used for tracking or correlation
+     */
+    requestId?: string | null;
+    /**
+     * Identifier for the schema being used or referenced
+     */
+    schemaId: string;
+    /**
+     * The block number on the blockchain where this announcement was recorded
+     */
+    blockNumber: number;
+    announcement: TombstoneAnnouncement | BroadcastAnnouncement | ReplyAnnouncement | ReactionAnnouncement | ProfileAnnouncement | UpdateAnnouncement;
+};
```

Comment:
Nit: formatting

Reply:
Odd. It's a generated file, but the generator claims to run the output through `prettier` if you configure it that way. Ran the formatter manually for now.